spec/menu_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'menu'
+
+describe Menu do
+
+  let(:menu) { Menu.new }
+  let(:dish) {double :dish}
+
```

Comment:
If you give the dish a name and a price you can use the same dish for all your tests.